### PR TITLE
Get Deploy Lag Badger to post to the repo owner's slack channel.

### PR DESCRIPTION
**Pull request by PeterHattyar**
https://trello.com/c/08crsjv6/3013-get-the-deploy-lag-badger-posting-to-the-owners-slack-channel-2

Made adjustments to get badger to post to repo owners' channels.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -50,42 +50,45 @@ end
...
+    message_payload = {
+      username: "Badger",
+      icon_emoji: ":badger:",
+      text: message,
+      mrkdwn: true,
+      channel: team,
+    }
+
+    puts
+    puts "Message for #{team}:"
```
Comment (MuriloDalRi) [inline Rakefile:74]: Are these two lines needed?

**Comment by MuriloDalRi** [comment]
The changes look good to me, have you tested if they work?

File: Rakefile (hunk 1 above)
Comment (PeterHattyar) [reply Rakefile:74]: Yeah we thought with James they should be there otherwise the user can’t know which channel each message is getting posted to.

Comment (PeterHattyar) [reply Rakefile:74]: And yes we ran the script and it works as intended.

**Review by MuriloDalRi — APPROVED** [review]

**Pull request merged**